frameworks/hdfs/universe/config.json

```diff
@@ -64,25 +64,25 @@
 				"secure_mode": {
 					"id": "http://hdfs/dcos/mesosphere.com/service/secure_mode",
 					"type": "boolean",
-					"description": "Enable HDFS Secure Mode",
+					"description": "Enable HDFS Secure Mode, which allows configuration of Kerberos.",
```

Comment:
It doesn't "allow configuration", it actually does the configuration.  "This enables Kerberos authentication, as well as network encryption for data and RPC calls."